Makefile

```diff
@@ -16,7 +16,7 @@ cross: clean
 	CGO_ENABLED=0 gox \
 	  -output="$(OUT_DIR)/jb-{{.OS}}-{{.Arch}}" \
 	  -ldflags=$(LDFLAGS) \
-	  -arch="amd64 arm64 arm" -os="linux" \
```

Comment:
Unsure about that one, don't think we need `windows/arm*` binaries? /cc @metalmatze 

Probably sufficient to add `-osarch="windows/amd64"`

Reply:
agreed

Reply:
I've removed the windows arm binaries in the latest commit